docs/docs/spec/metrics/metrics-prom.md

```diff
@@ -7,20 +7,38 @@ id: metrics-prom-spec
 
 The table below defines metrics which may be captured by AVS Nodes which expose metrics to Prometheus. AVSs may expose additional metrics however these should not use the `eigen_` prefix. 
 
+## Economics metrics
+
+| Name | Metric Type | Definition | Labels |
+|---|---|---|---|
+| `eigen_fees_earned_total` | Counter | The amount of fees earned in `<unit>` of underlying `<token>` in the `<strategy>` contract. This metric **SHOULD** be omitted while fees are yet to be implemented. | `token`, `unit`, `strategy` |
+| `eigen_slashing_status` | Gauge | Slashing status. The value **MUST** be 1 if the operator running `avs` has been slashed. This metric **SHOULD** not be implemented if there is no slashing in the target network (like in the upcoming testnet launch). | `avs` |
+| `eigen_registered_stakes` | Gauge | Operator stakes in AVS registry contract. Most commonly represents a weighted combination of delegated shares in the `DelegationManager` Eigenlayer contract. | `quorum` |
```

Comment:
change this to quorumNumber and quorumName

Reply:
Done [4d8b0c1](https://github.com/NethermindEth/el-node-spec/pull/58/commits/4d8b0c10138bcb4ffe3dce6bc6c667a6a3859750)